build/config.props

```diff
@@ -65,6 +65,10 @@
   <PropertyGroup Condition=" '$(Version)' == '' ">
     <Version>$(SemanticVersion)$(PreReleaseInformationVersion)</Version>
   </PropertyGroup>
+  
+  <PropertyGroup Condition=" '$(NuGetVsVersion)' == '' ">
+    <NuGetVsVersion>$(NuGetSdkVsSemanticVersion)$(PreReleaseInformationVersion)</NuGetVsVersion>
```

Comment:
I would have used a name like `VsSdkPackageVersion`, as we only use this version for the 2 (3?) Client SDK packages with VS extensibility APIs. NuGet's VSIX, and all other NuGet.* dlls are versioning with our normal 6.x, not 17.x version number.